engine/src/main/scala/cromwell/engine/io/RetryableRequestSupport.scala

```diff
@@ -0,0 +1,75 @@
+package cromwell.engine.io
+
+import java.io.IOException
+import java.net.{SocketException, SocketTimeoutException}
+
+import com.google.cloud.storage.StorageException
+import cromwell.engine.io.gcs.GcsBatchFlow.BatchFailedException
+import javax.net.ssl.SSLException
+
+trait RetryableRequestSupport {
```

Comment:
This feels more like a utility (or package?) `object` than a mix-in trait to me (the `class ConcreteRetry extends RetryableRequestSupport` in the spec was the pattern that rang some warning bells).

I think you could still use the `import RetryableRequestSupport._` pattern to avoid having to change all of the call sites.